### PR TITLE
Mark java package as shipped in APIView

### PR DESCRIPTION
**Pull request by praveenkuttappan**
Send a request to mark Java package as shipped in APIView after publishing the package.

### CODE REVIEW

**Review by weshaggard — APPROVED** [review]

File: eng/pipelines/templates/stages/archetype-java-release-batch.yml [hunk 1]
```diff
@@ -256,6 +256,13 @@ stages:
                     parameters:
                       ArtifactName: ${{parameters.ArtifactName}}-javadevfeed-$(System.JobAttempt)
                       ArtifactPath: $(Pipeline.Workspace)/${{parameters.ArtifactName}}-signed
+                  - template: /eng/common/pipelines/templates/steps/create-apireview.yml
+                    parameters:
+                      ArtifactPath: $(Pipeline.Workspace)/${{parameters.ArtifactName}}
```
Comment (praveenkuttappan) [inline eng/pipelines/templates/stages/archetype-java-release-batch.yml:261]:
```suggestion
                      ArtifactPath: $(Pipeline.Workspace)/${{parameters.ArtifactName}}-signed
```

File: eng/pipelines/templates/stages/archetype-java-release-batch.yml [hunk 2]
```diff
@@ -256,6 +256,13 @@ stages:
...
+                      Artifacts: ${{parameters.Artifacts}}
+                      ConfigFileDir: $(Pipeline.Workspace)/${{parameters.ArtifactName}}/PackageInfo
```
Comment (praveenkuttappan) [inline eng/pipelines/templates/stages/archetype-java-release-batch.yml:263]:
```suggestion
                      ConfigFileDir: $(Pipeline.Workspace)/${{parameters.ArtifactName}}-signed/PackageInfo
```

File: eng/pipelines/templates/stages/archetype-java-release-batch.yml [hunk 3]
```diff
@@ -256,6 +256,13 @@ stages:
...
+                      MarkPackageAsShipped: true
+                      ArtifactName: ${{parameters.ArtifactName}}
```
Comment (praveenkuttappan) [inline eng/pipelines/templates/stages/archetype-java-release-batch.yml:265]:
```suggestion
                      ArtifactName: ${{parameters.ArtifactName}}-signed
```

File: eng/pipelines/templates/stages/archetype-java-release.yml [hunk 4]
```diff
@@ -242,6 +242,15 @@ stages:
                           ArtifactName: ${{parameters.ArtifactName}}-${{artifact.name}}-javadevfeed-$(System.JobAttempt)
                           ArtifactPath: $(Pipeline.Workspace)/${{parameters.ArtifactName}}-signed
 
+                      - template: /eng/common/pipelines/templates/steps/create-apireview.yml
+                        parameters:
+                          ArtifactPath: $(Pipeline.Workspace)/${{parameters.ArtifactName}}
```
Comment (praveenkuttappan) [inline eng/pipelines/templates/stages/archetype-java-release.yml:247]:
```suggestion
                          ArtifactPath: $(Pipeline.Workspace)/${{parameters.ArtifactName}}-signed
```

File: eng/pipelines/templates/stages/archetype-java-release.yml [hunk 5]
```diff
@@ -242,6 +242,15 @@ stages:
...
+                          Artifacts: ${{parameters.Artifacts}}
+                          ConfigFileDir: $(Pipeline.Workspace)/${{parameters.ArtifactName}}/PackageInfo
```
Comment (praveenkuttappan) [inline eng/pipelines/templates/stages/archetype-java-release.yml:249]:
```suggestion
                          ConfigFileDir: $(Pipeline.Workspace)/${{parameters.ArtifactName}}-signed/PackageInfo
```

File: eng/pipelines/templates/stages/archetype-java-release.yml [hunk 6]
```diff
@@ -242,6 +242,15 @@ stages:
...
+                          MarkPackageAsShipped: true
+                          ArtifactName: ${{parameters.ArtifactName}}
```
Comment (praveenkuttappan) [inline eng/pipelines/templates/stages/archetype-java-release.yml:251]:
```suggestion
                          ArtifactName: ${{parameters.ArtifactName}}-signed
```

**Review by JimSuplizio — APPROVED** [review]
As per our discussion, the changes are good.

**Comment by praveenkuttappan** [comment]
This change was tested by releasing azure-template packages using pipeline run https://dev.azure.com/azure-sdk/internal/_build/results?buildId=3795795&view=results

**Pull request merged**